tools/autocoders/templates/ac_network_defs.h

```diff
@@ -8,8 +8,9 @@
 #define _AUTOCODER_NETWORK_DEFS_H_
 
 #include <stdint.h>
+#include <string>
 
-#define CREATE_IP_ADDR(base, octet1, octet2) STRINGIFY(base) "." STRINGIFY(octet3) "." STRINGIFY(octet4)
+#define CREATE_IP_ADDR(base, octet3, octet4) std::string(base) + "." + std::to_string(octet3) + "." + std::to_string(octet4)
```

Comment:
Can this just be a contexpr function since were not interested in autocoding for C? Since were not using STRINGIFY anymore and assuming something that can construct a string for `base` and to_string-able (probably int) types for `octet3` and `octet4` you can avoid macros and get types and better error messages

Reply:
It can't. I've tried a ton of different things to try and make it constexpr and std::to_string is not constexprable in C++20

Reply:
that's so annoying. @bjarne figure this out